client/src/pages/CreateNewPage.css

```diff
@@ -0,0 +1,46 @@
+.Form {
+}
+
+.Page__heading {
+  border: 2px none black;
+  padding: 2em 8em;
+  border-radius: 15px;
+  background: #f4b7b7;
+  display: flex;
+  align-items: center;
+  color: #d84064;
+  margin: 3em;
+  overflow-y: auto;
+}
+#name {
```

Comment:
try to use BEM here like you did very well before. :)
also even if this is one more property try to use classes.
to give the text more space I tend to use padding instead of height, this makes it even more easy to maintain in case you change font-sizes.